src/Components/Impact.js

```diff
@@ -33,6 +33,50 @@ export default class Impact extends Component {
                 para2: "right way!",
               }}
             />
+          </Row>
+          <Row
+            className="justify-content-md-center"
+            style={{ marginTop: "5px" }}
+          >
```

Comment:
@Meghag5 The below component is repeated multiple times, can't we have a map over a JSON?
```js
<ImpactCard
   impact={{
       name: "Open-Source Journey",
       imgUrl: "image3.svg",
       para1:
          "Get your journey in open-source started with our open-source projects!",
                para2: "",
    }}
 />
```